src/types/redis_json.cc

```diff
@@ -159,11 +159,14 @@ rocksdb::Status Json::Type(const std::string &user_key, const std::string &path,
   auto s = read(ns_key, &metadata, &json_val);
   if (!s.ok()) return s;
 
-  auto res = json_val.Type(path, results);
+  auto res = json_val.Type(path);
+  if (!res) return rocksdb::Status::InvalidArgument(res.Msg());
```

Comment:
Would it be an invalid value here? Rather than just InvalidArgument

Reply:
It will return error status iff the `path` is invalid.

Reply:
Oh I see, when type is unknown it will just return a unknown string